### PR TITLE
Update Venmo icon and colour

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Venmo

**Issue:** n/a
**Alexa rank:** [~2.8k](https://www.alexa.com/siteinfo/venmo.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [ ] I optimized the icon with SVGO or SVGOMG
  - [ ] The SVG `viewbox` is `0 0 24 24`

### Description
Colour taken from base-64 encoded SVG in [website](https://venmo.com/) header. I also changed the source to the website as [the old source](https://venmo.com/about/brand/) looks to be outdated, but we should monitor it for updates.

### CODE REVIEW

**Comment by jorgeamadosoria** [comment]
Why is the color different in the header than in the brand guidelines?
How are we decide which one takes precedence?

**Comment by adamrusted** [comment]
I would say the new hex is sufficient. As @PeterShaggyNoble says, it appears the brand guidelines are outdated - as their app icons use the new colour. In that case, we should follow the brand's lead and use the _new_ colour.

Would it also be worth us considering changing to just the `v` in this PR?

**Comment by jorgeamadosoria** [comment]
> I would say the new hex is sufficient. As @PeterShaggyNoble says, it appears the brand guidelines are outdated - as their app icons use the new colour. In that case, we should follow the brand's lead and use the _new_ colour.
> 
> Would it also be worth us considering changing to just the `v` in this PR?

I think the full wordmark is fine, but if we are going to change to just the v, I think it should be a white v over a blue background, like in their favicon.

**Comment by adamrusted** [comment]
Updated to the app icon (had to construct it myself from the PNG found [here](https://venmo.com/about/brand/).

venmo

**Review by service-paradis — CHANGES_REQUESTED** [review]

File: icons/venmo.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Venmo</title><path d="M4.78 0v.016A5 5 0 0 0 .011 4.838H0v14.324h.012a5 5 0 0 0 4.767 4.822V24h14.325v-.008a5 5 0 0 0 4.884-4.83H24V4.838h-.004A5 5 0 0 0 19.104.002V0H4.779zm12.296 5.88c.467.742.66 1.482.66 2.442 0 3.046-2.607 6.998-4.72 9.797H8.213L6.263 6.54l4.227-.412 1.016 8.232c.96-1.536 2.14-3.978 2.14-5.652 0-.906-.165-1.537-.412-2.059l3.842-.767z"/></svg>
```
[the remark this answers is not in the document]
Comment (adamrusted) [reply icons/venmo.svg:1]: You're right. I'd not properly read through this. I'll update _again_ to the full wordmark based on the SVGs at that link..

**Review by service-paradis — APPROVED** [review]
LGTM 🎉

**Pull request merged**